src/newgrf_gui.cpp

```diff
@@ -57,7 +57,11 @@ void ShowNewGRFError()
 		for (uint i = 0; i < lengthof(c->error->param_value); i++) {
 			SetDParam(4 + i, c->error->param_value[i]);
 		}
-		ShowErrorMessage(STR_NEWGRF_ERROR_FATAL_POPUP, INVALID_STRING_ID, WL_CRITICAL);
+		if (c->error->severity == STR_NEWGRF_ERROR_MSG_FATAL) {
+			ShowErrorMessage(STR_NEWGRF_ERROR_FATAL_POPUP, INVALID_STRING_ID, WL_CRITICAL);
+		} else {
+			ShowErrorMessage(STR_NEWGRF_ERROR_POPUP, INVALID_STRING_ID, WL_CRITICAL);
```

Comment:
I would guess we shouldn't fire this as `WL_CRITICAL`. `WL_ERROR` sounds more appropriate in this case?